src/Helper/NumberHelper.php

```diff
@@ -0,0 +1,348 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\IntlBundle\Helper;
+
+use Sonata\IntlBundle\Locale\LocaleDetectorInterface;
+
+/**
+ * NumberHelper displays culture information.
+ *
+ * @author Thomas Rabaix <thomas.rabaix@ekino.com>
+ * @author Stefano Arlandini <sarlandini@alice.it>
+ */
+final class NumberHelper extends BaseHelper
+{
+    /**
+     * @var array The default attributes to apply to the \NumberFormatter instance
+     */
+    protected array $attributes = [];
```

Comment:
There are some protected things that should be private, given the new classes are final.